proto/sonic_internal.proto

```diff
@@ -34,4 +34,7 @@ message Value {
 
   // Notification to be used in place of 1-4 if present
   gnmi.Notification notification = 7;
-}
\ No newline at end of file
+
+  // Delete to be used to indicate that node was deleted
+  []gnmi.Path delete = 8;
```

Comment:
This is an incorrect syntax. It should have been `repeated gnmi.Path delete = 8`. I opened PR https://github.com/sonic-net/sonic-gnmi/pull/157 to fix.

Please do not update the sonic_internal.pb.go file manually when the proto file was modified. Run `make proto/sonic_internal.pb.go` to re-generate it, which will report all syntax errors in the proto file.